libpod/runtime.go

```diff
@@ -162,6 +162,8 @@ type RuntimeConfig struct {
 	InfraImage string `toml:"infra_image"`
 	// InfraCommand is the command run to start up a pod infra container
 	InfraCommand string `toml:"infra_command"`
+	//LabelSupport indicates wether libpod will support container labeling
+	LabelSupport bool `toml:"label"`
```

Comment:
This doesn't have any business in libpod except to piggyback on the config file - it affects absolutely nothing below config generation.

Reply:
I know we don't want another config file, but this really does not belong here. Can we get away with an environment variable?

Reply:
Read the issue.  I was thinking this could be used to override selinux=true label in crio.conf also.

Reply:
I'm still not in favor - it's adding something that is not used at all by libpod and we have to rely on upper layers to implement. If we want to keep the config flag here we should also handle clearing the SELinux labels in libpod as part of NewContainer

Reply:
While I've done the same in #1500, I do see @mheon point here. Clearing the labels in NewContainer is probably a better approach. Something like this should do the trick

```
diff --git a/libpod/runtime_ctr.go b/libpod/runtime_ctr.go
index a0b576bc..d5afb924 100644
--- a/libpod/runtime_ctr.go
+++ b/libpod/runtime_ctr.go
@@ -77,6 +77,13 @@ func (r *Runtime) newContainer(ctx context.Context, rSpec *spec.Spec, options ..
                ctr.config.Namespace = r.config.Namespace
        }
 
+       if !r.config.LabelSupport {
+               ctr.config.Spec.Process.SelinuxLabel = ""
+               if ctr.config.Spec.Linux != nil {
+                       ctr.config.Spec.Linux.MountLabel = ""
+               }
+       }
+
        for _, option := range options {
                if err := option(ctr); err != nil {
                        return nil, errors.Wrapf(err, "error running container create option")
```